content/zh/docs/access-control-and-account-management/external-authentication/use-an-ldap-service.md

```diff
@@ -16,42 +16,37 @@ weight: 12220
 
```

Comment:
@Felixnoo  We have a special case here. LDAP was broken in kubesphere v3.2.0. we need to let users change the ks-apiserver image when config LDAP.
`kubectl -n kubesphere-system set image deployment/ks-apiserver *=kubespheredev/ks-apiserver:release-3.2`
/cc @FeynmanZhou

Reply:
Will this bug be fixed in v3.2.1?

Reply:
Yes. It would be fixed in v3.2.1 .